arviz/data/inference_data.py

```diff
@@ -10,6 +10,7 @@
 from html import escape
 
 import netCDF4 as nc
+import json
```

Comment:
We should try to import first in try-except

    import ujson as json

There are actually couple of drop-in placements for json. Should we have a function that decides which of the libs is used?